docs/content/users/extend/additional-services.md

```diff
@@ -73,11 +76,11 @@ Anyone can create an add-on for `ddev get`. See [this screencast](https://www.yo
 
 The `install.yaml` is a simple YAML file with a few main sections:
 
-* `pre_install_actions`: an array of Bash statements or scripts to be executed before `project_files` are installed. The actions are executed in the context of the target project’s root directory.
-* `project_files`: an array of files or directories to be copied from the add-on into the target project’s `.ddev` directory.
-* `global_files`: is an array of files or directories to be copied from the add-on into the target system’s global `.ddev` directory (`~/.ddev/`).
-* `post_install_actions`: an array of Bash statements or scripts to be executed after `project_files` and `global_files` are installed. The actions are executed in the context of the target project’s root directory.
-* `yaml_read_files`: a map of `name: file` of YAML files to be read from the target project’s root directory. The contents of these YAML files may be used as templated actions within `pre_install_actions` and `post_install_actions`.
+- `pre_install_actions`: an array of Bash statements or scripts to be executed before `project_files` are installed. The actions are executed in the context of the target project’s root directory.
```

Comment:
Are you changing us to dashes for bullets? If so, that should be added to the markdown rules for markdownlintcheck

Reply:
Nope! Must have absentmindedly let VS Code format the Markdown instead of paying attention to the linter. Will fix.

Reply:
[Swapped them back](https://github.com/drud/ddev/pull/4296/commits/9cc2b37e6e20d24ff124798456088307e0b4ed5a), thank you!